modules/manage/pages/iceberg/redpanda-topics-iceberg-snowflake-catalog.adoc

```diff
@@ -15,14 +14,14 @@ This guide walks you through querying Redpanda topics as Iceberg tables in https
 * xref:manage:tiered-storage.adoc#configure-object-storage[Object storage configured] for your cluster and xref:manage:tiered-storage.adoc#enable-tiered-storage[Tiered Storage enabled] for the topics for which you want to generate Iceberg tables.
 ** The S3 bucket URI so that you can configure it as external storage for Open Catalog.
```

Comment:
```suggestion
** You need the S3 bucket URI, so you can configure it as external storage for Open Catalog.
```

Reply:
@kbatuigas if you have time, I'd tidy up the Prereqs so that these extra lines (like here and other prereqs) just sit nested under their item but not as a list item.